src/pages/TransactionHistory/components/DatePicker/DatePicker.css

```
@@ -109,6 +109,7 @@
   background-color: var(--contentBackground);
   margin:0;
   padding:0.4rem;
+  height:380px;
```

```suggestion
  min-height:380px;
```

Extra safety in case it would require more height in the future 🙏